api/src/controllers/files.ts

```diff
@@ -117,6 +118,20 @@ export const multipartHandler: RequestHandler = (req, res, next) => {
 	}
 };
 
+router.get(
+	'/scan',
+	asyncHandler(async (req, res, next) => {
+		const storageFolder = storage.disk('local');
```

Comment:
We can't rely on a hardcoded storage adapter named `local`. `local` is just the default name, but seeing its user created, we can't rely on that name alone